spec/services/submission/plant_trial_zip_exporter_spec.rb

```diff
@@ -8,7 +8,7 @@
 
   describe "#call" do
     it 'compress three files in the zip file' do
-      csv_data = Submission::PlantTrialZipExporter.new.call(plant_trial).read
+      csv_data = Submission::PlantTrialZipExporter.new.call(plant_trial, 'no_cache').read
```

Comment:
Passing `'no_cache'` seems a bit misleading, since it does not disable cache. It is reused unless cleared between subsequent test runs by some other means. We should probably disable cache globally in test environment (e.g. by switching to `NullStore`) and enable it only for these cases which actually test caching.